evergreen.yml

```
@@ -166,3 +214,9 @@ buildvariants:
     tasks:
       - name: compile
       - name: test-rust
+
+  - name: ubuntu1804
+    display_name: Ubuntu Sanitizers
+    run_on: [ ubuntu1804-test ]
```

```suggestion
  - name: ubuntu2004
    display_name: Ubuntu 20.04
    run_on: [ ubuntu2004-large ]
```
a couple comments:
- Is there any reason not to use the newest ubuntu?
- I don't think there's any need to have a sanitizer-specific variant -- that's a property of the task, not the variant -- if we start building the driver on Linux in the future, it can be on this variant too